lang/well-formed.md

```diff
@@ -186,7 +187,10 @@ impl ValueExpr {
                         Type::Int(IntType { signed: Unsigned, size: M::PTR_SIZE })
                     }
                     Int2Ptr(ptr_ty) => {
-                        ensure(matches!(operand, Type::Int(IntType { signed: Unsigned, size: M::PTR_SIZE })))?;
```

Comment:
Sadly one cannot match like this `size: M::PTR_SIZE`.

Reply:
Does `Type` have equality? Then we could just use `==` instead of `matches!`?

Reply:
Yep, thats better!